tensorflow/contrib/distributions/python/kernel_tests/bijectors/affine_test.py

```diff
@@ -461,13 +462,14 @@ def dynamic_run(fun, x_value, **kwargs):
   def testNoBatchMultivariateRaisesWhenSingular(self):
     with self.cached_session():
       mu = [1., -1]
-      bijector = Affine(
-          shift=mu,
-          # Has zero on the diagonal.
-          scale_diag=[0., 1],
-          validate_args=True)
-      with self.assertRaisesOpError("diagonal part must be non-zero"):
-        bijector.forward([1., 1.]).eval()
+      with self.assertRaisesRegexp(errors.InvalidArgumentError,
+                                   "diagonal part must be non-zero"):
+        bijector = Affine(
+            shift=mu,
```

Comment:
Unused variable 'bijector' [unused-variable]

Reply:
Changed `bijector` to `_`.